### DISCUSSION TITLE
記事本文がサイドバーより短いとフッターがサイドバーに埋まっている

### DISCUSSION THREAD

**Discussion by takkanm** [rubima/magazine.rubyist.net#417] [記事についての報告]
環境は Microsoft Edge バージョン 112.0.1722.58 (公式ビルド) (64 ビット) 。他の環境では未確認です。

![スクリーンショット 2023-04-30 171453](https://user-images.githubusercontent.com/43500/235342978-7a8acae1-7c1a-4353-97e4-115e55192aa0.png)

**Comment by snoozer05** [comment]
Chromeの 112.0.5615.137（Official Build）（arm64）では大丈夫そうですねえ。

<img width="1544" alt="スクリーンショット 2023-05-02 6 57 36" src="https://user-images.githubusercontent.com/3954/235538902-6efea9ad-d873-49e7-b625-f2d10dea7851.png">

Safariのバージョン16.4 (18615.1.26.11.23)でも問題なさそうでした。
確認に使ったページは次の2つです。

https://magazine.rubyist.net/articles/pretokyorubykaigi11/preTokyoRubyKaigi11-index.html
https://magazine.rubyist.net/articles/0052/0052-present-feedback.html

**Comment by takkanm** [comment]
フッターの文字列はセンタリングで表示されており、メインコンテンツの div の幅が 1200 ぐらいあるとちょうど収まるが、それより小さいと埋まっていくという感じのようです。
構造は以下のような感じなので、footer のほうもコンテナわけてあげるとかですかねぇ。
```html
<div class="row full">
  <div class="col-md-2 hidden-xs sidbar>...</div>
  <div class="col-md-10 main">...</div>
</div>
<footer>
  <div class="container"> ... </div>
</footer>
```

**Comment by snoozer05** [comment]
> フッターの文字列はセンタリングで表示されており、メインコンテンツの div の幅が 1200 ぐらいあるとちょうど収まるが、それより小さいと埋まっていくという感じのようです。

なるほど。再現できました。

> 構造は以下のような感じなので、footer のほうもコンテナわけてあげるとかですかねぇ。

単純にコンテナを分けただけでは現象は解決しませんでした。
サイドバーのスタイルが影響してそう。
issue化して対応考えますね
